spec/integration/post_show_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
## Requirement :memo:

![image](https://user-images.githubusercontent.com/93039359/225404825-080ec149-3429-456b-9845-fe2b988d9e2c.png)

## Logs Screenshot

![image](https://user-images.githubusercontent.com/93039359/225404188-90297889-1de2-43da-b631-8a6bec958943.png)

- [ ] It would be nice if you add logs screenshots in your pull request before and after solving N+1 problem to follow the requirements :blush:

:red_circle: **Question**
How can i get the logs?
:green_circle: **Answer**

- Run your server using `rails server`
- Open the **user_post_index** and **user_post_show** because you need to add two logs

Now you can see logs in the VS Code terminal :+1: